test/libgit2.jl

```diff
@@ -1905,7 +1905,8 @@ mktempdir() do dir
                         deserialize(f)
                     end
                     @test err.code == LibGit2.Error.ECERTIFICATE
-                    @test startswith(err.msg, "The SSL certificate is invalid")
+                    @test startswith(lowercase(err.msg),
```

Comment:
if libgit2 has changed case convention in their error messages and this wasn't showing up with mbedtls then I guess we should find where in https://github.com/libgit2/libgit2/pull/4173 this error is happening and leave a comment that it should be made consistent